.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+auth_key
+network
```

Comment:
Should there be a newline after this? git seems to give me a little warning icon. Not sure if it matters.

Reply:
I'm on Windows, newline seems not affecting me.

But if warning is on your machine, I'll put newline in for good.

Reply:
No I just meant when reviewing the code on the Github web site there's a little warning icon. I'll merge it anyway. If it's a problem for unix we'll fix it later. :)